src/css/style.css

```diff
@@ -96,13 +96,28 @@ body {
 }
 
 .container {
-    width: 100vw;
+    max-width: 1000px;
+    margin: 0 auto;
+}
 
-    display: flex;
-    justify-content: center;
+@media only screen and (max-width: 600px) {
+    .container {
+        padding: 15px;
+    }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)